src/timeMachine/reducers/fluxDocsReducer.ts

```diff
@@ -0,0 +1,20 @@
+// Types
+import {Fluxdocs} from 'src/client/fluxdocsdRoutes'
+import {Action} from 'src/timeMachine/actions/scriptEditorThunks'
+
+export interface FluxDocsState {
+  fluxDocs: Fluxdocs[]
+}
+
+const INITIAL_STATE: FluxDocsState = {
+  fluxDocs: [],
+}
+
+export default (state = INITIAL_STATE, action: Action): FluxDocsState => {
+  switch (action.type) {
+    case 'GET_FLUX_DOCS':
```

Comment:
After changing to the new pattern mentioned in the first comment above, you can import the `GET_FLUX_DOCS` constant to ensure consistent naming.